advanced_alchemy/extensions/litestar/plugins/init/plugin.py

```diff
@@ -118,5 +120,10 @@ def on_app_init(self, app_config: AppConfig) -> AppConfig:
             )
             app_config.before_send.append(cast("BeforeMessageSendHookHandler", config.before_send_handler))
         app_config.signature_namespace.update(signature_namespace_values)
+        if configure_exception_handler and not any(
```

Comment:
Only set the default handler for the `RepositoryError` when we've got a configuration flag to do so and there is not an existing handler for `RepositoryError`